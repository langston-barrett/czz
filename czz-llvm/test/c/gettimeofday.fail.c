#include <stdio.h>
#include <sys/time.h>
int main(int argc, char *argv[]) {
  struct timeval t;
  gettimeofday(&t, NULL);
  printf("tv_sec: %ld\ntv_usec: %ld", t.tv_sec, t.tv_usec);
  return 0;
}
