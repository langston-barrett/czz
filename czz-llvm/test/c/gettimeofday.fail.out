tv_sec: 1661711109
tv_usec: 626529